packages/plugins/integration-zapier/src/lib/zapier.config.ts

```
@@ -4,8 +4,16 @@ export const ZAPIER_API_URL = 'https://api.zapier.com/v1';
+export const ZAPIER_CALLBACK_URL = `${process.env['API_BASE_URL']}/api/integration/zapier/oauth/callback`;
+
 /** URL for obtaining OAuth access tokens from Gauzy */
 export const ZAPIER_TOKEN_URL = `${process.env['API_BASE_URL']}/api/integration/zapier/token/:integrationId`;
 
 /** URL for redirecting users after OAuth authorization */
 export const ZAPIER_REDIRECT_URI = 'https://zapier.com/dashboard/auth/oauth/return/App221848CLIAPI/';
+
+/** OAuth token expiration time in seconds (1 hour) */
+export const ZAPIER_TOKEN_EXPIRATION_TIME = 3600;
+
+console.log('Zapier Config -> ZAPIER_API_URL: ' + process.env);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)